### ISSUE TITLE
Sway crashes randomly

### ISSUE THREAD

**Issue by DarkDeepBlue** [swaywm/sway#1035]
Sway seems crashes without being responsive. Although this cannot be reproduced reliable, it happens quite often and might be related to switching or moving workspaces.
Switching TTYs is not possible. The log file doesn’t contain any useful information.

The output configuration is as following:
```
output LVDS-1 resolution 1600x900  position 0,0
output VGA-1  resolution 1600x1200 position 0,-1200
```

In the kernel logs the following is found:
```
Jan 07 13:08:15 titan kernel: traps: sway[4135] general protection ip:41ecb7 sp:3fc5070d540 error:0 in sway[400000
Jan 07 13:08:15 titan systemd[1]: Created slice system-systemd\x2dcoredump.slice.
Jan 07 13:08:16 titan systemd[1]: Started Process Core Dump (PID 4214/UID 0).
Jan 07 13:08:16 titan systemd-coredump[4215]: Process 4135 (sway) of user 1000 dumped core.

Stack trace of thread 4135:
#0  0x000000000041ecb7 close_views (sway)
#1  0x000000000040f9c1 cmd_kill (sway)
#2  0x000000000040be40 handle_command (sway)
#3  0x000000000042324d n/a (sway)
#4  0x000003d59e1ea2a4 n/a (libwlc.so.0)
#5  0x000003d59e1ed151 n/a (libwlc.so.0)
#6  0x000003d59e200a44 n/a (libwlc.so.0)
#7  0x000003d59d90ed32 wl_event_loop_dispatch (libwayland-server.so.
#8  0x000003d59d90d1da wl_display_run (libwayland-server.so.0)
#9  0x000003d59e201e2c wlc_run (libwlc.so.0)
#10 0x000000000040abcd main (sway)
#11 0x000003d59c163291 __libc_start_main (libc.so.6)
#12 0x000000000040ad1a _start (sway)
```

+ System: ArchLinux x64 4.8.16 grsec
+ GCC 6.2.1 20160830

+ Sway: baa84a0
+ Wayland: 1.12.0-1
+ WLC: 61dbcf2

**Comment by fhsjaagshs** [comment]
I just encountered this bug myself, and I submitted an issue without reading yours. Oops. It's issue 1036.

**Comment by SirCmpwn** [comment]
Yours look like different bugs to me.

**Comment by SirCmpwn** [comment]
I'm closing issues that are resolved by the yet-unreleased sway 1.0.

**Comment by Merrick-919** [comment]
My one just closes on its own sometimes.
